listenbrainz/webserver/static/js/src/personal-recommendations/PersonalRecommendationsModal.tsx

```diff
@@ -0,0 +1,263 @@
+        data-backdrop="static"
+      >
+        <div className="modal-dialog" role="document">
+          <form className="modal-content">
+            <div className="modal-header">
+              <button
+                type="button"
+                className="close"
+                data-dismiss="modal"
+                aria-label="Close"
+                onClick={() => {
+                  this.setState({
+                    users: [],
+                    blurbContent: "",
+                    suggestions: [],
+                  });
+                  // update the followers list
+                  APIService.getFollowersOfUser(currentUser.name)
+                    .then((response) => {
+                      this.setState({ followers: response.followers });
+                    })
+                    .catch((error) => {
+                      this.handleError(error, "Error while fetching followers");
+                    });
+                }}
+              >
+                <span aria-hidden="true">&times;</span>
+              </button>
+              <h4 className="modal-title" id="PersonalRecommendationModalLabel">
+                Personally recommend this recording
+              </h4>
+            </div>
+            <div className="modal-body">
+              {users!.map((user) => {
+                return (
+                  <NamePill
+                    title={user}
+                    // eslint-disable-next-line react/jsx-no-bind
+                    closeAction={this.removeUser.bind(this, user)}
+                    closeButton
+                  />
+                );
+              })}
+              <input
+                type="text"
+                className="form-control"
+                onChange={this.searchUsers}
+                placeholder="Search and add users"
+              />
+              <SearchDropDown
+                suggestions={suggestions}
+                // eslint-disable-next-line react/jsx-no-bind
+                action={this.addUser}
+              />
+              <p style={{ marginTop: "24px", marginBottom: "24px" }}>
+                Tell the users selected above why are you recommending them{" "}
+                <b>
+                  {" "}
+                  {track_name} by {artist_name}
+                </b>
+                ? (Optional)
+              </p>
+              <div className="form-group">
+                <textarea
+                  className="form-control"
+                  id="blurb-content"
+                  placeholder="Let your followers know why you recommended this song"
+                  value={blurbContent}
+                  name="blurb-content"
+                  rows={4}
+                  style={{ resize: "vertical" }}
+                  spellCheck="false"
+                  onChange={this.handleBlurbInputChange}
+                />
+              </div>
+              <small style={{ display: "block", textAlign: "right" }}>
```

Comment:
Since you already have a .less file defined for this component, I'd like to see all the `style={…` in this file moved to LESS definitions instead. It's much easier to keep things up to date when the style are defined in only one place.